### PR TITLE
Fix javadoc generation.

### PR DESCRIPTION
**Pull request by johnou**

### CODE REVIEW

**Comment by kuujo** [comment]
This still doesn't generate the aggregated Javadocs... only the docs for each module.

**Comment by johnou** [comment]
@kuujo what if you use the javadoc:aggregate mojo?

**Comment by kuujo** [comment]
Should be fine now

File: protocols/raft/src/main/java/io/atomix/protocols/raft/ReadConsistency.java [hunk 1]
```diff
@@ -15,10 +15,12 @@
  */
 package io.atomix.protocols.raft;
 
+import io.atomix.protocols.raft.protocol.QueryRequest;
+
 /**
- * Constants for specifying Raft {@link RaftQuery} consistency levels.
+ * Constants for specifying Raft {@link QueryRequest} consistency levels.
```
Comment (kuujo) [inline protocols/raft/src/main/java/io/atomix/protocols/raft/ReadConsistency.java]: Should probably use `OperationType#QUERY`. `QueryRequest` is an internal API

**Pull request merged**